elements/pf-checkbox/demo/validation.html

```diff
@@ -0,0 +1,10 @@
+<pf-checkbox label="Required" required></pf-checkbox>
```

Comment:
The `required` attribute is not showing the asterisk for me, Chrome Version 117.0.5938.149 (Official Build) (arm64)

Reply:
Working now